composer.json

```diff
@@ -24,8 +24,8 @@
 	},
 	"require": {
 		"php": ">=5.3.0",
-		"wikibase/data-model": "~2.5",
-		"wikibase/data-model-serialization": "~1.1",
+		"wikibase/data-model": "~3.0",
+		"wikibase/data-model-serialization": "~1.4-dev",
```

Comment:
Not sure this works. If you want to modify the stability, then the syntax is `@modifier`. The branch alias for master is `1.4.x-dev`, though I'm not sure usage of this will be allowed without also decreasing the min stability.